docs/lib/govuk_tech_docs/open_api/renderer.rb

```diff
@@ -122,11 +122,12 @@ def schemas_from_schema(schema)
         schemas
       end
 
-      def operations(path, path_id)
+      def operations(text, path, path_id)
```

Comment:
Is it potentially worth passing these args in as a hash now given two of the params are strings?

Reply:
or keyword args?